src/sections/Blog/Blog-sidebar/blogSidebar.style.js

```diff
@@ -165,8 +165,54 @@ const BlogSideBarWrapper = styled.div`
        
     }
    
-      
-       
+    .explain-1 {
+        padding-top: 0rem;
+        text-align: center;
+        .cards {
+            margin: 0.15rem auto 0 ;
+            max-width: 50rem;
+            padding: 1.5rem 1rem 0rem 1rem;
+            background-color: none;
+            border-radius: 25px;
+            .card {
+                -webkit-transition: 450ms all;
+                transition: 450ms all;
+                margin: 2px;
+                padding: 1.25rem;
+                background-color: #1E2117; 
```

Comment:
@debo19 could you please remove this hard-coded color. I think this is dark Jungle green, please use that from the themes.